### PR TITLE
chore(eslint): integrate eslint

### PR DESCRIPTION
**Pull request by obulaworld**
#### What does this PR do?
-Integrate ESlint to the project

#### Description of Task to be completed?
- Integration of eslint as a linting tool to make sure the right coding conventions are used.

#### How should this be manually tested?
- Deos not need a test

#### What are the relevant pivotal tracker stories?
[Configure ESLInt](https://www.pivotaltracker.com/story/show/160451588 "Configure ESLint")

### CODE REVIEW

**Review by Mrshinezee — APPROVED** [review]
LGTM

**Comment by andela-phemmy** [comment]
LGTM

**Review by andela-phemmy — APPROVED** [review]
LGTM

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      }],
+      "require-jsdoc": ["error", {
+          "require": {
+              "FunctionDeclaration": true,
+              "MethodDefinition": true,
+              "ClassDeclaration": true
+          }
+      }]
+    }
+};
```
Comment (KaiserPhemi) [inline .eslintrc.js]: #### Nitpick
Insert new line at the end of file.

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (obulaworld) [reply .eslintrc.js]: Okay.On it now.
Thanks

Comment (obulaworld) [reply .eslintrc.js]: Added the new line to .eslintrc.js

**Review by KaiserPhemi — APPROVED** [review]

**Pull request merged**